emailing-service/emailing-service-test/obj/rider.project.restore.info

```diff
@@ -1 +1 @@
-﻿17279680392729411
\ No newline at end of file
+﻿17281671575055597
```

Comment:
I also don't know what this is.. could you explain ? Thank you

Reply:
Once again, it's in the obj folder so it's just a thing with people building the project on their computer and pushing it. I didn't write that

Reply:
Ok, I see